specs/DesignDiscussions/Inconsistencies.md

```diff
@@ -0,0 +1,179 @@
+* Features that are easy to implement on one platform might be quite difficult to implement on another. This can sometimes lead to compromised implementations that end up shipping without being brought to spec, or sometimes features that just don't get implemented across all platforms (!!)
+  * This can also present a form of schedule risk because if it so happens that the last platform ends up being the hardest, you can end up with more work than calendar
+* Type coercion in Javascript can cause it to be more permissive than other platforms leading to some constructs like `maxlines: "2"` being valid in JS, but invalid in, say, UWP (if JS code isn't carefully written -- we do, of course, author in Typescript, but we do also have to keep these problems in mind)
+* Distinct implementations means that defaults can vary by platform (e.g. whether the `TextBlock` `wrap` property defaults to `true` if not specified)
+* Differing choices made by devs at implementation time can lead to divergent behaviors (e.g. whether an invalid property value is silently dropped vs. emitting a warning vs. failing to parse the card)
+* Supporting libraries and platform affordances available to us can ease or speed up implementation, but can also be the source of inconsistencies (e.g. differences in how a dev constructs .NET's `DateTime`, UWP's `DateTime`, C++'s `std::chrono::time_point`, and Javascript's `Date` -- instantiating each object with, for example, a string, can vary in permissivity)
+  * Indeed, we've had a few issues arise from differences between a hand-rolled solution and a pre-rolled solution (e.g. date/time handling (#1915) and support for markdown (#1984))
+
+### Rendering
+
+Rendering presents another avenue by which inconsistencies can sneak into Adaptive Cards. Some of these problems can be pretty tricky to solve.
+
+* Multiple implementations means that "gotcha" scenarios might not always be covered or may be covered inconsistently (e.g. what happens when you combine `maxLines`, markdown lists, and `wrap: true`?)
+* Platform-provided controls can vary widely in their behaviors and support. Even if divergent behavior is caught during implementation, it may be difficult or tricky to solve completely
+
+## Proposals
+
+* (P0) Change processes to prevent new inconsistencies from ever happening:
+  * Motivation: Inconsistencies are caught before they ship
+  * Tasks: 
+    * New bugs automatically filed with `inconsistency` tag, which is only removed once it's determined that the issue is *not* a consistency issue
+    * We can't ship if an `inconsistency`-tagged bug hasn't been resolved for this release
+    * Features that need implementation across all platforms are merged to a single branch and PR'd/tested for inconsistencies before being allowed to merge upstream
+      * Each feature is championed by a single dev or PM, who owns the feature from conception to first ship
```

Comment:
>feature [](start = 13, length = 7)

Discussion in the room: More specific discussion of what counts as a "feature" for this process. We don't need feature branches for platform specific features, bugs, etc.